x-pack/metricbeat/module/aws/rds/_meta/data.json

```diff
@@ -0,0 +1,68 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "aws": {
+        "rds": {
+            "cpu": {
+                "total": {
+                    "pct": 3
+                }
+            },
+            "database_connections": 0,
+            "db_instance": {
+                "arn": "arn:aws:rds:us-east-2:627959692251:db:test1",
+                "class": "db.r5.large",
+                "identifier": "test1",
+                "status": "available"
+            },
+            "deadlocks": 0,
+            "disk_usage": {
+                "bin_log.bytes": 0
+            },
+            "free_local_storage": 32977604608,
```

Comment:
it seems this has `.bytes` suffix in `fields.yml`? I wonder why this is not detected by tests

Reply:
Yeah good catch, I fixed the `.bytes` problem. The test `TestFetch` didn't catch it because I didn't add this field into the `mtest.CheckEventField`. Also `TestFetch` is integration test that requires aws credentials in CI which we don't have.
The reason why I'm not spending time here is: we have a separate ticket to solve this issue for aws module https://github.com/elastic/beats/issues/11226